### PR TITLE
Add more test for CdCommand for test coverage

### PR DESCRIPTION
**Pull request by anthony6401**

### CODE REVIEW

**Review by Ma-Yueran — APPROVED** [review]
Looks good to merge!

File: src/test/java/seedu/address/logic/commands/CommandTestUtil.java [hunk 1]
```diff
@@ -32,6 +34,7 @@
     public static final String VALID_FILE_ADDRESS_CS2101 = "./src/test/java/seedu/address/testutil/cs2101.bat";
     public static final String VALID_MAC_FILE_ADDRESS_CS2101 = "./src/test/java/seedu/address/testutil/cs2101.sh";
     public static final String VALID_MAC_FILE_ADDRESS_TESTFILE = "./src/test/java/seedu/address/testutil/testFile.sh";
+    public static final String VALID_CHILD_FOLDER = "child";
```
Comment (luo-git) [inline src/test/java/seedu/address/logic/commands/CommandTestUtil.java]: Will this be really valid for CdCommand?

**Review by luo-git — APPROVED** [review]
Looks good. 👍 Coding style wise, maybe we shouldn't be using too many newlines.

File: src/test/java/seedu/address/logic/commands/CommandTestUtil.java (hunk 1 above)
Comment (anthony6401) [reply src/test/java/seedu/address/logic/commands/CommandTestUtil.java]: I'm just using it as cd user input, but yeah I think the name is kinda inaccurate. I'll change it to something that the project has!

**Review by luo-git — APPROVED** [review]
LGTM!

**Pull request merged**